ote_cli/ote_cli/utils/hpo.py

```diff
@@ -232,6 +232,13 @@ def run_hpo_trainer(
         else:
             hyper_parameters.learning_parameters.num_iters = hp_config["iterations"]
 
+    # add batch size value if batch size isn't in search space
```

Comment:
this comment should be changed like
"current HPO requires batch_size information so if batch_size is not in the HPO's target hparams, need to set batch_size config from the instance of ConfigurableParameters"

Reply:
I see. I modified as you said.